### DISCUSSION TITLE
关于map 类型

### DISCUSSION THREAD

**Discussion by v1xingyue** [NonceGeek/Move-Camp#182] [Q&A] [answered]
为什么Move中没有map类型呢？
似乎Move 中符合类型只有 vector 一种吧。
如果需要 map 类型，有没有相应的实现思路呢？

**Comment by MagicGordon** [comment, marked as answer]
可以参考aptos源码：
1、https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/sources/table.move
2、https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/sources/table_with_length.move
3、https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/sources/simple_map.move
